ciphey/basemods/Crackers/XandY.py

```diff
@@ -0,0 +1,108 @@
+# community
+# by https://github.com/lukasgabriel
+
+from distutils import util
+from typing import Optional, Dict, Union, Set, List
+from loguru import logger
+
+from ciphey.iface import ParamSpec, Cracker, CrackResult, CrackInfo, T, registry, Config
+
+
+@registry.register
+class XandY(Cracker[str]):
+    def getInfo(self, ctext: str) -> CrackInfo:
+        # TODO Write something useful here
+        return CrackInfo(
+            success_likelihood=1, success_runtime=1e-5, failure_runtime=1e-5,
```

Comment:
I don't think its success likelihood of 1 if it can return nothing? not sure on how this works tbh

Reply:
For this, I didn't know what to put so I tried different values as I was trying to fix another bug. I'll change it to something more reasonable.

Reply:
If it checks two different variants, would the success likelihood be 0.5?

Reply:
I'll leave it at 0.1 for now. Feel free to change it though!